config/routes.rb

```diff
@@ -58,6 +58,18 @@
     end
   end
 
+  namespace :api do
+    namespace :v1 do
+      namespace :admin do
+        resources :admins, only: %i[]  do
+          collection do
+            get '/active_users', to: 'users#active_index'
+          end
+        end
+      end
+    end
+  end
+
```

Comment:
You shouldn't re-create the whole `api - v1` namespace. Just move it under the existing one

Reply:
> You'll prob want to put `/active_users` under the users controller

I'm confused, it's already in the users admin controller ?